.travis.yml

```diff
@@ -0,0 +1,23 @@
+dist: trusty
+sudo: false
+language: node_js
+notifications:
+  email: false
+node_js:
+  - "8"
+  - "6"
+cache:
+  directories:
+    - node_modules
+script:
+  - "npm run lint"
```

Comment:
Only lint for now - adding tests & stuff in https://github.com/snyk/snyk-paket-parser/pull/3 and we'll update this to include `npm run test` there